test/integration/unuthenticated_user_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.